src/Repository/InvoiceRepository.php

```diff
@@ -89,4 +97,23 @@ public function getFilteredPagination(InvoiceFilterData $invoiceFilterData, int
             ['defaultSortFieldName' => $defaultSortField, 'defaultSortDirection' => 'desc']
         );
     }
+
+    private function getProjectIdsFromViewId(int $viewId): array
+    {
+        $view = $this->entityManager->getRepository(View::class)->find($viewId);
```

Comment:
Use this instead: https://github.com/itk-dev/economics/blob/develop/src/Service/ViewService.php#L23
Then you don't have to use viewId in parameter.